win32/initnet.c

```diff
@@ -4,12 +4,13 @@
 #include <crtdbg.h>
 #endif
 
+#include <winsock2.h>
 #include <windows.h>
 
-#ifdef DLL_EXPORT
-
```

Comment:
this is bogus, winlogin.c will compile finding the variable but will fail running as the resources (rc file) needed is not included.
Better to fail compiling and fixing the project files.

Reply:
I'll take another look, thanks.